sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -963,8 +963,11 @@ ProgramManager::ProgramPtr ProgramManager::build(
                                         DeviceLibReqMask);
   }
 
+  static const char *ForceLinkEnv = std::getenv("SYCL_FORCE_LINK");
```

Comment:
@againull, @smaslov-intel, shouldn't this variable be documented in https://github.com/intel/llvm/blob/sycl/sycl/doc/EnvironmentVariables.md#debugging-variables-for-dpc-runtime section?

Reply:
I added this environment variable to help the device driver team reproduce two bugs that I reported.  Since I expect to remove the environment variable after those bugs are fixed, I didn't bother adding documentation.  If people want, I can add documentation now, but we will just remove it again once the bugs are fixed.

Reply:
I am OK to add this to experimental section